<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 232, 232);
</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>521</width>
     <height>401</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Nazwisko i Imie</string>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data wprowadzenia</string>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kwota Kredytu</string>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
    <property name="flags">
     <set>ItemIsEditable</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="font">
     <font>
      <family>Artifakt Element</family>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 242);
border-color: rgb(73, 84, 100);
border: 2px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Szukaj...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>Usuń z bazy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>500</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 84, 100);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Cofnij</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rozpoczęci klienci</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 242);</string>
   </property>
   <item>
    <property name="text">
     <string>Wybierz z listy...</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 242);</string>
   </property>
   <item>
    <property name="text">
     <string>Wybierz z listy...</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Sfinalizuj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 242);</string>
   </property>
   <item>
    <property name="text">
     <string>Wybierz z listy...</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 190);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Oblicz prowizję</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
